.github/workflows/ci.yml

```diff
@@ -81,11 +81,17 @@ jobs:
         uses: "./"
         with:
           channel: "${{ matrix.release-env }}"
+          disable-metrics: true
+
 
       - name: "Test: flox --version"
         run: |
           flox --version
 
+      - name: "Test: flox config"
+        run: |
+          flox config
```

Comment:
Can we make this a test that checks if disable-metrics is set correctly in config?

Reply:
Sure.